lib/resque/plugins/timed_round_robin/timed_round_robin.rb

```diff
@@ -36,10 +41,6 @@ def slice_expired?
       Time.now > @slice_expiration
     end
 
-    def begin_new_queue(queue)
-
-    end
```

Comment:
Does @bschmeck remember why he added this empty method 2 years ago? It looks like an oversight as part of a refactor. https://github.com/KennaSecurity/resque-timed-round-robin/commit/089b091b65b55b093a26edbca9b097cdc6073a57#diff-003ca568a2e162aedd40c999145c9ea8R35

Reply:
I meant to add a note about this, I also assumed it was leftover dead code so I removed it

Reply:
I have no idea what I was doing there.  Almost certainly dead code.